Tests/TestingApps/PaywallsTester/PaywallsTester/Views/UpsellView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LockedView.swift
+//  PaywallsTester
+//
+//  Created by Andrés Boedo on 9/28/23.
+//
+
+import SwiftUI
+
+struct UpsellView: View {
+    var body: some View {
+        VStack {
+            Text("""
+                This view automatically displays the default paywall if you're not subscribed.
+                
+                This is achieved by calling
+                `.presentPaywallIfNeeded(requiredEntitlementIdentifier: \(Configuration.entitlement))
```

Comment:
Love it. Small typo:
```suggestion
                This is achieved by calling
                `.presentPaywallIfNeeded(requiredEntitlementIdentifier: \(Configuration.entitlement))`
```